data/json/items/book/traps.json

```diff
@@ -52,9 +52,8 @@
   {
     "id": "textbook_traps",
     "type": "BOOK",
-    "name": "The Compleat Trapper",
-    "name_plural": "The Compleat Trapper",
-    "//": "The spelling is intentionally archaic, as this form is commonly used in the titles of books.",
```

Comment:
I originally thought it was too until I saw the comment and did a quick google search to learn more